extensions/amp-embedly-card/validator-amp-embedly-card.protoascii

```diff
@@ -14,17 +14,33 @@
 # limitations under the license.
 #
 
-tags: {  # embedly-card
+tags: {  # embedly-card 1.0
   html_format: AMP
   tag_name: "SCRIPT"
+  satisfies: "amp-embedly-card 1.0"
+  excludes: "amp-embedly-card 0.1"
   extension_spec: {
     name: "amp-embedly-card"
+    version_name: "v1.0"
+    version: "1.0"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # embedly-card 0.1 and latest
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-embedly-card 0.1"
+  excludes: "amp-embedly-card 1.0"
+  extension_spec: {
+    name: "amp-embedly-card"
+    version_name: "v0.1"
     version: "0.1"
     version: "latest"
+    requires_usage: EXEMPTED
+    deprecated_allow_duplicates: true
```

Comment:
This should not be added if not already present, since these are looser validations than are necessary.
```suggestion
```